make/compiler_flags

```diff
@@ -117,10 +117,36 @@ CPPFLAGS_SUNDIALS ?= -DNO_FPRINTF_OUTPUT $(CPPFLAGS_OPTIM_SUNDIALS) $(CXXFLAGS_F
 
 ## setup compiler flags
 CXXFLAGS_LANG ?= -std=c++1y
-#CXXFLAGS_BOOST ?=
 CXXFLAGS_SUNDIALS ?= -pipe $(CXXFLAGS_OPTIM_SUNDIALS) $(CPPFLAGS_FLTO_SUNDIALS)
 #CXXFLAGS_GTEST
 
+ifdef BOOST_INC
+DEPS_VERSION_CHECK ?= true
+CXXFLAGS_BOOST ?= -I $(BOOST_INC)
+else
+CXXFLAGS_BOOST ?= -I $(BOOST)
+endif
+
+ifdef SUNDIALS_LIB
+LDFLAGS_SUNDIALS ?= -L $(SUNDIALS_LIB) -lsundials_cvodes -lsundials_idas -lsundials_kinsol -lsundials_nvecserial
+ifdef SUNDIALS_INC
+DEPS_VERSION_CHECK ?= true
+CXXFLAGS_SUNDIALS += -I $(SUNDIALS_INC)
+endif
+else
+CXXFLAGS_SUNDIALS += $(INC_SUNDIALS)
+endif
+
+ifdef SUNDIALS_INTERFACE_OLD
```

Comment:
??? what's `SUNDIALS_INTERFACE_OLD`?

this looks like some testing that got included.